src/DocsGenerator/Helpers/LevelHelper.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using Codacy.Engine.Seed.Patterns;
+
+namespace Codacy.TSQLLint.DocsGenerator.Helpers
+{
+    /// <summary>
+    /// Helper to convert to codacy level from a TSQLLint rule
+    /// </summary>
+    public static class LevelHelper
+    {
+        /// <summary>
+        /// This map a rule to a codacy level.
+        /// </summary>
+        /// <param name="rule">rule name</param>
+        /// <returns>Codacy level</returns>
+        /// <exception cref="NotImplementedException">When found a non explicit mapped rule</exception>
+        public static Level ToLevel(string rule)
+        {
+            switch (rule)
+            {
+                case "keyword-capitalization":

```

Comment:
shouldn't this come from a configuration file instead to be hard-coded?

Reply:
This is to map the category to our `Level`, and the configuration file was `error` by default. On this tool, unfortunately, we need to do it manually.